lizmap/modules/lizmap/controllers/service.classic.php

```diff
@@ -315,36 +315,39 @@ protected function getServiceParameters()
         }
 
         // Optionally filter data by login
-        if (isset($params['request'])) {
+        /*if (isset($params['request'])) {
```

Comment:
I think commented code should be removed.

Reply:
Yes, the PR is not finished yet.

Reply:
Done